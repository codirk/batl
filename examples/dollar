This template shows a batl bug:

   $ <--- ${echo -n just a dollar sign}
