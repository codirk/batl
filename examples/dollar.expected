This template shows a batl bug:

   $ <--- just a dollar sign
